docs/apache-airflow/installation/upgrading.rst

```diff
@@ -79,6 +79,20 @@ table or rename it or move it to another database using those tools. If you don'
 can use the ``airflow db shell`` command - this will drop you in the db shell tool for your database and you
 will be able to both inspect and delete the table.
 
+How to drop the table using Kubernetes:
+
+
+1. Exec into any of the Airflow pods - webserver or scheduler: ``kubectl exec -it <your-webserver-pod> python``
+
+2. Run the following commands in the python shell:
+
+ .. code-block:: python
+   from airflow.settings import Session
+
+   session = Session()
+   session.execute("DROP TABLE _airflow_moved__2_2__task_instance")
+   session.commit()
+
```

Comment:
```suggestion
 .. code-block:: python
     from airflow.settings import Session

     session = Session()
     session.execute("DROP TABLE _airflow_moved__2_2__task_instance")
     session.commit()

```

I _think_ indenting by two spaces is not enough. Could be wrong, but let’s be safe, IIRC GitHub and Sphinx also have slightly different behaviours.